packages/web/pages/dashboard/new-post.tsx

```diff
@@ -15,10 +15,14 @@ import {
   useNewPostQuery,
   useCreatePostMutation,
   PostStatus as PostStatusType,
+  PostsQuery,
+  PostsQueryVariables,
+  PostsDocument,
 } from '@/generated/graphql'
 import AuthGate from '@/components/AuthGate'
 import { useTranslation } from '@/config/i18n'
 import useUILanguage from '@/hooks/useUILanguage'
+import { toast } from 'react-toastify'
```

Comment:
Let's put this library import at the top

Reply:
created the pr #501 following the approach that I've been using so far